test/iterative.jl

```diff
@@ -6,7 +6,7 @@
         @testset "5-bus case" begin
             result = PowerModelsRestoration.run_iterative_restoration(data, PowerModels.ACPPowerModel, juniper_solver, repair_periods=3)
 
-            @test result["termination_status"] == LOCALLY_SOLVED
+            @test result["termination_status"] == PowerModels.LOCALLY_SOLVED
             @test isapprox(result["objective"], 1899.5; atol = 1e0)
             @test isapprox(length(keys(result["solution"]["nw"])), 9; atol=1e-1)
 
```

Comment:
Adding this `PowerModels.` in the tests is acceptable, but I tend to be a little more lazy and do "using PowerModels" or "using MathOptInterface" in `runtests.jl` to bring these constants into the global scope.

I'll leave the final choice up you.

Reply:
I thought there was push to remove `using` from tests and only using `import`?

Reply:
I will leave it as `PowerModels.` for now, as this is used similarly for  the formulations `PowerModels.DCPPowerModel`.

Reply:
At least for now, it has been a matter of preference.  I prefer `using` in tests David tends to prefer `import`.

The only hard rule is `import` only inside packages, in the tests it is more flexible.